diff --git a/build/configuration-list.js b/build/configuration-list.js
index 3a2a4c124ca58cae8813640dffe0667a98b69e39..64d5304df1a45c157b20cfea0348a353c3531d20 100644
--- a/build/configuration-list.js
+++ b/build/configuration-list.js
@@ -674,6 +674,7 @@ function getConfigurationListBuildSettingsForType(project, props) {
         case "bg-download":
         case "credentials-provider":
         case "device-activity-monitor":
+        case "device-activity-report":
         case "intent":
         case "intent-ui":
         case "location-push":
diff --git a/build/target.js b/build/target.js
index bc7f3f5551674af5ffbd9363e8c449b530bb5263..89bd28321459efca034f34e21d21f33a0b8cfbb2 100644
--- a/build/target.js
+++ b/build/target.js
@@ -116,6 +116,12 @@ exports.TARGET_REGISTRY = {
         frameworks: ["DeviceActivity"],
         displayName: "Device Activity Monitor",
     },
+    "device-activity-report": {
+        extensionPointIdentifier: "com.apple.deviceactivityui.report-extension",
+        productType: "com.apple.product-type.extensionkit-extension",
+        frameworks: ["DeviceActivity", "SwiftUI", "FamilyControls"],
+        displayName: "Device Activity Report",
+    },
     "network-packet-tunnel": {
         extensionPointIdentifier: "com.apple.networkextension.packet-tunnel",
         needsEmbeddedSwift: true,
